pzp/pzp.py

```diff
@@ -87,9 +89,13 @@ def initGui(self):
         toolButton.setPopupMode(QToolButton.MenuButtonPopup)
         self.toolbar.addWidget(toolButton)
 
-        action = self.create_action("help.png", "Aiuto", self.do_help)
-        self.toolbar.addAction(action)
-        self.iface.addPluginToMenu(PLUGIN_NAME, action)
+        settings_action = self.create_action("gear.png", "Ipostazioni", self.do_settings)
```

Comment:
Italian typo 😛 

(My level of Italian has suddenly increased by 0.0000001%)

Reply:
good catch thanks :)